hyperscale/Dockerfile

```diff
@@ -0,0 +1,70 @@
+# This file is auto generated from it's template,
+# see citusdata/tools/packaging_automation/templates/docker/latest/latest.tmpl.dockerfile.
+FROM postgres:14.0
+ARG VERSION=10.2.2
+LABEL maintainer="Citus Data https://citusdata.com" \
+      org.label-schema.name="Citus" \
+      org.label-schema.description="Scalable PostgreSQL for multi-tenant and real-time workloads" \
+      org.label-schema.url="https://www.citusdata.com" \
+      org.label-schema.vcs-url="https://github.com/citusdata/citus" \
+      org.label-schema.vendor="Citus Data, Inc." \
+      org.label-schema.version=${VERSION} \
+      org.label-schema.schema-version="1.0"
+
+ENV CITUS_VERSION ${VERSION}.citus-1
+ENV POSTGRES_DB citus
+
+# install Citus
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+       ca-certificates \
+       curl \
+    && curl -s https://install.citusdata.com/community/deb.sh | bash \
+    && apt-get install -y postgresql-$PG_MAJOR-citus-10.2=$CITUS_VERSION \
+                          postgresql-$PG_MAJOR-hll=2.16.citus-1 \
+                          postgresql-$PG_MAJOR-topn=2.4.0 \
+    && apt-get purge -y --auto-remove curl \
+    && rm -rf /var/lib/apt/lists/*
+
+# install other extensions
+RUN     apt-get update \
+      &&apt-get install -y postgresql-14-postgis-3 \
+            postgresql-plperl-14 \
+            postgresql-contrib \
+            postgresql-14-partman \
+            postgresql-14-tdigest \
+            postgresql-14-mysql-fdw
+
+RUN apt-get -y install postgresql-14-cron
```

Comment:
I suggest that you combine all the package installation steps in here.

Reply:
Combined with the upper command